src/ui/globals/constants.ts

```diff
@@ -1,17 +1,11 @@
 // @TODO - foconnor: Some of these could live closer to the component and some could come from the env.
-const MNEMONIC_FIFTEEN_WORDS = 15;
-const MNEMONIC_TWENTYFOUR_WORDS = 24;
-const FIFTEEN_WORDS_BIT_LENGTH = 160;
-const TWENTYFOUR_WORDS_BIT_LENGTH = 256;
+const MNEMONIC_SIXTEEN_WORDS = 16;
```

Comment:
imo we don't need this constant anymore since the UI isn't generating the seed phrase anymore. It can just be a local variable in the tests where it's used (and should be 18 - so I guess the component can handle an arbitrary long seed phrase, which is good)